### ISSUE TITLE
How to use google TTS provider?

### ISSUE THREAD

**Issue by gabrielschuck** [duncan3dc/speaker#7]
I tried here on my local server to follow the quick example of using the google text-to-speech service, but I'm getting error 403 ...

**Comment by duncan3dc** [comment]
@gabrielschuck Can you provide some more details? Maybe a short script to re-produce the error?

**Comment by duncan3dc** [comment]
@gabrielschuck Also could you try out this test version to see if it fixes your issue:
```sh
composer require duncan3dc/speaker:dev-google-fix
```

**Comment by gabrielschuck** [comment]
Hi, the code example I tried was what's in the readme.

```
Fatal error : Uncaught exception 'GuzzleHttp\Exception\ClientException'
with message 'Client error: GET http://translate.google.com/translate_t ts?q=+Hello+World+&tl=en&client=duncan3dc%2Fspeaker resulted in a 403 Forbidden response: ' in C:\xampp\htdocs\tradutor\vendor\guzzlehttp\guz
zle\src\Exception\RequestException.php:111 Stack trace: #0
C:\xampp\htdocs\tradutor\vendor\guzzlehttp\guzzle\src\Middleware.php(65)
: GuzzleHttp\Exception\RequestException::create(Object(GuzzleHttp\Psr7\R
equest), Object(GuzzleHttp\Psr7\Response)) #1
C:\xampp\htdocs\tradutor\vendor\guzzlehttp\promises\src\Promise.php(203)
: GuzzleHttp\Middleware::GuzzleHttp{closure}(Object(GuzzleHttp\Psr7\Res
ponse)) #2 C:\xampp\htdocs\tradutor\vendor\guzzlehttp\promises\src\Promi
se.php(156): GuzzleHttp\Promise\Promise::callHandler(1,
Object(GuzzleHttp\Psr7\Response), Array) #3
C:\xampp\htdocs\tradutor\vendor\guzzlehttp\promises\src\TaskQueue.php(47
): GuzzleHttp\Promise\Promise::GuzzleHttp\Promise{closure}() #4
C:\xampp\htdocs\tradutor\vendor\guzzlehttp\prom in
C:\xampp\htdocs\tradutor\vendor\guzzlehttp\guzzle\src\Exception\RequestE
xception.php on line 111
```

**Comment by duncan3dc** [comment]
This is fixed in [0.7.3](https://github.com/duncan3dc/speaker/releases/tag/0.7.3)
